ckanext/dia_theme/fanstatic/banner.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+(function ($) {
+  var showBanner = function (content) {
+    if (content) {
+      $('header').after(content)
+    }
+  }
+
+  var getBannerHost = function () {
+    var local = window.location.host.startsWith('ckan-gateway')
+    if (local) {
+      return 'http://ckan-gateway:8000'
+    }
+
+    var production = window.location.host.startsWith('catalogue')
+    return production ? '//data.govt.nz' : '//diadata-uat.cwp.govt.nz'
+  }
+
+  var getBanner = function () {
+    $.ajax({
+      url: getBannerHost() + '/api/banner',
+      success: function (banner) {
```

Comment:
probably should add an error handler here too, will update and just console log the error if one is received... otherwise we'll get them coming through raygun I expect.